test/rest_api/test_compaction_task.py

```diff
@@ -82,6 +83,7 @@ def test_major_keyspace_compaction_task(cql, this_dc, rest_api):
     check_compaction_task(cql, this_dc, rest_api, lambda keyspace, table: rest_api.send("POST", f"column_family/major_compaction/{keyspace}:{table}?flush_memtables=true"), "major compaction", task_tree_depth)
     check_compaction_task(cql, this_dc, rest_api, lambda keyspace, table: rest_api.send("POST", f"column_family/major_compaction/{keyspace}:{table}?flush_memtables=false"), "major compaction", task_tree_depth)
 
+@pytest.mark.xfail(reason="rest_api suite doesn't support tablets yet (#17338), run test manually")
```

Comment:
I'm afraid I don't understand again. If in the previous patch you added skip_with_tablets, why don't you use it here instead of this xfail?

Reply:
I don't use it here, because I assumed `skip_{with,without}_tablets` is good for the cases when the test is tablet-specific (or vnodes-specific) and doesn't make sense with the other. 

On the contrary, I think xfail is intended to be a quick, temporary "fix" waiting to be replaced with something better. So, together with enabling tablets by default, I added xfails to prevent CI failures. I didn't add all the actual fixes in this commit, because I think each test file deserves its own fix-commit and squashing them would make a mess

Reply:
Same as above.